### ISSUE TITLE
Discussion: Workflow for Doc-Block Changes

### ISSUE THREAD

**Issue by kkrieger85** [OpenMage/magento-lts#1038]
There are[ 20 open PR regarding Doc-Blocks](https://github.com/OpenMage/magento-lts/pulls?q=is%3Apr+is%3Aopen+label%3A%22Cleanup%3A+DOC+blocks%22)

I propose to **not discuss every single unclear change** in a whole bunch of changes (one PR with e.g. up to 80 files). 

Why: 
- It's only doc-block. It doesn't break code.
- It blocking these changes

How to make it better:
- Check, if all changes are doc-block only and merge PR
- Block PR only if code is affected
- After that, you/other could start a new issue for a single doc-block which seems to be wrong or unclear.

**Comment by tmotyl** [comment]
Hi
I second the idea of getting things done and I have reviewed all the doc block patches (some multiple times) and merged some.
However I disagree with loosing quality standards.

1) doc blocks are important(of course not as important as code changes but still), and these changes as all others should follow quality standards of the code review
2) in some of the patches already we have identified the issues which require further discussion or fixing in the code (analyze of docblocks lead to finding bugs). For this cases we’re opening new issues not to block the patches. Without reviewing them we would not be able to identify the issues.
3) all the patches already have one positive review so one more we get and the patch can be merged
4) so if you and few other contributors could revie one patch a day we will be soon done.
So I can only encourage you to get get some reviewes done. This woukd be highly appreciated
Thanks
